packages/packages.json

```diff
@@ -1,38 +1,38 @@
+        "skill/valory/test_abci/0.1.0": "bafybeigr6jaxbbnk4vuepm7kkhfkk6qbpbxj2i6apujzyi4obxuf2ttd7y",
+        "agent/valory/abstract_abci/0.1.0": "bafybeifscxhijvsl7jdkty6aqnsziahcz6glumtlwggjzrtjescdol4j3m",
+        "agent/valory/counter/0.1.0": "bafybeidbpguhrd5nu7axca7pgnk76fjyozhvjnxz2gb546kmjqbk6exgiu",
         "agent/valory/counter_client/0.1.0": "bafybeiammfm2m3xatutqrn6xxp7tty3bzynqjqwjjiygezvcrbbnrf62o4",
-        "agent/valory/hello_world/0.1.0": "bafybeicbnl4eeq5dzmrgv4hkv4inahqh3edjksus7x5nqasjieladbbeui",
-        "agent/valory/register_reset/0.1.0": "bafybeicpykcdp2nujq7y3pbyjwelpxzoglwfq6fxqg7mkalswrc45n3j6i",
-        "agent/valory/register_termination/0.1.0": "bafybeidyvb7qjexppp6qi7odhvelk6ezpdysbi726oe55y4tpag35eusv4",
-        "agent/valory/registration_start_up/0.1.0": "bafybeif6bsppopbvt3yffnjmlhqnourratpruhuhfcrbopzya2uitar5ja",
-        "agent/valory/test_abci/0.1.0": "bafybeiayxoqwv642jl5p56hq24muhdtvn3ey4nvpvp2msmlio3zqhq5vtu",
-        "service/valory/counter/0.1.0": "bafybeih6vru2ljabqdrvkwu2p3n2fj36e6mgwkusv6tmsd6nho7si5oc2a",
-        "service/valory/hello_world/0.1.0": "bafybeifcw2nlmoh7rh6pasxsdyhdwv4gj3qu34egs5l3i4b6wgrlo7gsvm",
-        "service/valory/register_reset/0.1.0": "bafybeifjvcviwfnp5wqdiq6awuwc5yrtkvcnt5pweii7lxbdgdo3bykxuq",
-        "skill/valory/safe_deployment_abci/0.1.0": "bafybeih7hesj4rulosmmvp6qhxyunxczih5pjol2qzvzou63aujetjlcly",
-        "skill/valory/register_reset_recovery_abci/0.1.0": "bafybeiduushkkap3tf7xm4gq7rgge5lvncsybap37vaowkqtif274e34sm",
-        "agent/valory/register_reset_recovery/0.1.0": "bafybeigjcvhvpzgxkvuhvhgzjasb52fyvv5qko7amxbtyld2gbbjugshfi"
+        "agent/valory/hello_world/0.1.0": "bafybeiey7zsejahaobwpcynuuki2xxwlolq3iolhgypowpeeunmysxaary",
+        "agent/valory/register_reset/0.1.0": "bafybeieusmfxhru3sgto7tyiqiz7fmcqmvdrcgagvdk5h6n44dbzkoutwi",
+        "agent/valory/register_termination/0.1.0": "bafybeidqfavmehntwfsjwzksz7kyjpcx4mfeidttjcgfgzrn2k3byl34wm",
+        "agent/valory/registration_start_up/0.1.0": "bafybeief2p5ngx5siudg55bjq3z6iuvtaojlnegqwkihgodtiywdfueiy4",
+        "agent/valory/test_abci/0.1.0": "bafybeig2s5uiomakecuzcfm76icvkqr7xdmywm4h54oildppj73xe2oe24",
+        "service/valory/counter/0.1.0": "bafybeihi6l2zaqbacykstg45qftonkvmuxrwbevxh4emrvwl4qsnqi6xhi",
+        "service/valory/hello_world/0.1.0": "bafybeidkly5wgeopyxom5bwhtrq57ckfei7x6zavi4bowaranijutpefde",
+        "service/valory/register_reset/0.1.0": "bafybeiaxkrt7c47nqretkhw6mexqkbxibbdjn7z6vooxqrzcui2witrnqy",
+        "skill/valory/safe_deployment_abci/0.1.0": "bafybeidfueovwewv47g2juk6dzgit3rvzxur34gqqb2ghq56vjfv56h7ji",
+        "skill/valory/register_reset_recovery_abci/0.1.0": "bafybeihfzm2czimnq5nmotaxr2rvciuzb6smnjzom5thsdiftzazqskena",
+        "agent/valory/register_reset_recovery/0.1.0": "bafybeicdo6drxrgptldfg3vu6hgarrzeybfz2yfapb2oehxzyvipofhrge"
     },
     "third_party": {
```

Comment:
Please confirm we are up-to-date wrt to these. I had the same question on the PR re 1.26 of open-aea

Reply:
We haven't had a change in hashes on open-aea for past month